2016-11-03 21:36:43,252 | INFO     | log.py               (128) | setupLogging | Choosing RAET log level 1
2016-11-03 21:36:45,592 | DEBUG    | testable.py          (126) | decorator | in SpyableTestClient added spy on __init__
2016-11-03 21:36:45,600 | WARNING  | testable.py          (134) | decorator | method <function Client.handleOneNodeMsg at 0x7f7279688ea0> not found, so no spy added
2016-11-03 21:36:45,627 | DEBUG    | testable.py          (126) | decorator | in SpyableTestMonitor added spy on __init__
2016-11-03 21:36:45,628 | DEBUG    | testable.py          (126) | decorator | in SpyableTestMonitor added spy on isMasterReqLatencyTooHigh
2016-11-03 21:36:45,628 | DEBUG    | testable.py          (126) | decorator | in SpyableTestMonitor added spy on isMasterThroughputTooLow
2016-11-03 21:36:45,628 | DEBUG    | testable.py          (126) | decorator | in SpyableTestMonitor added spy on sendThroughput
2016-11-03 21:36:45,631 | DEBUG    | testable.py          (126) | decorator | in SpyableTestLedgerManager added spy on __init__
2016-11-03 21:36:45,631 | DEBUG    | testable.py          (126) | decorator | in SpyableTestLedgerManager added spy on catchupCompleted
2016-11-03 21:36:45,639 | DEBUG    | testable.py          (126) | decorator | in SpyableTestLedgerManager added spy on processConsistencyProofReq
2016-11-03 21:36:45,639 | DEBUG    | testable.py          (126) | decorator | in SpyableTestLedgerManager added spy on startCatchUpProcess
2016-11-03 21:36:45,642 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on __init__
2016-11-03 21:36:45,642 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on addToPrePrepares
2016-11-03 21:36:45,647 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on canProcessPrePrepare
2016-11-03 21:36:45,647 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on canSendPrepare
2016-11-03 21:36:45,647 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on doOrder
2016-11-03 21:36:45,647 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on doPrePrepare
2016-11-03 21:36:45,648 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on doPrepare
2016-11-03 21:36:45,648 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on isValidPrepare
2016-11-03 21:36:45,648 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on orderPendingCommit
2016-11-03 21:36:45,648 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on processCommit
2016-11-03 21:36:45,648 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on processPrePrepare
2016-11-03 21:36:45,648 | DEBUG    | testable.py          (126) | decorator | in SpyableTestReplica added spy on processPrepare
2016-11-03 21:36:45,661 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on __init__
2016-11-03 21:36:45,661 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on checkPerformance
2016-11-03 21:36:45,662 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on decidePrimaries
2016-11-03 21:36:45,662 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on discard
2016-11-03 21:36:45,662 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on eatTestMsg
2016-11-03 21:36:45,662 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on forward
2016-11-03 21:36:45,662 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on handleInvalidClientMsg
2016-11-03 21:36:45,662 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on handleOneNodeMsg
2016-11-03 21:36:45,662 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on postToClientInBox
2016-11-03 21:36:45,662 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on postToNodeInBox
2016-11-03 21:36:45,662 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on processInstanceChange
2016-11-03 21:36:45,663 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on processOrdered
2016-11-03 21:36:45,663 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on processPropagate
2016-11-03 21:36:45,663 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on processRequest
2016-11-03 21:36:45,671 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on propagate
2016-11-03 21:36:45,671 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on reportSuspiciousClient
2016-11-03 21:36:45,671 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on reportSuspiciousNode
2016-11-03 21:36:45,671 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on send
2016-11-03 21:36:45,672 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on sendInstanceChange
2016-11-03 21:36:45,672 | DEBUG    | testable.py          (126) | decorator | in SpyableTestNode added spy on startViewChange
Loading module /home/lovesh/.virtualenvs/sovrin-dev/lib/python3.5/site-packages/anoncreds_dev-0.1.8-py3.5.egg/config/config-crypto-example1.py
Module loaded.
2016-11-03 21:36:46,269 | DEBUG    | gen_load_ratcheting  (280) | main | Results file: /home/lovesh/dev/evernym/sovrin-priv/scripts/perf_results_1_2_1478189206.csv
2016-11-03 21:36:46,269 | INFO     | gen_load_ratcheting  (136) | _spawnClient | Seed for client Claaaa7 is b'0000000000000000000000000Claaaa7', its len is 32
2016-11-03 21:36:46,276 | DEBUG    | ledger.py            (164) | start | Starting ledger...
2016-11-03 21:36:46,276 | DEBUG    | ledger.py            (69) | recoverTree | Recovering tree from transaction log
2016-11-03 21:36:46,307 | INFO     | client.py            (121) | __init__ | Client Claaaa7 initialized with the following node registry:
2016-11-03 21:36:46,307 | INFO     | client.py            (128) | __init__ |     EvernymV1C      listens at 52.33.22.91    on port 9722
2016-11-03 21:36:46,307 | INFO     | client.py            (128) | __init__ |     EvernymV2C      listens at 52.38.24.189   on port 9724
2016-11-03 21:36:46,307 | INFO     | client.py            (128) | __init__ |     WSECUC          listens at 13.93.218.255  on port 9726
2016-11-03 21:36:46,307 | INFO     | client.py            (128) | __init__ |     BIGC            listens at 52.160.103.164 on port 9728
2016-11-03 21:36:46,307 | INFO     | client.py            (128) | __init__ |     RespectNetworkC listens at 54.173.9.185   on port 9730
2016-11-03 21:36:46,307 | INFO     | client.py            (128) | __init__ |     FCUC            listens at 69.144.146.171 on port 9734
2016-11-03 21:36:46,307 | INFO     | client.py            (128) | __init__ |     BYUC            listens at 54.193.6.155   on port 9745
2016-11-03 21:36:46,307 | DEBUG    | plugin_helper.py     (25) | loadPlugins | Plugin loading started to load plugins from basedir: /home/lovesh/.sovrin
2016-11-03 21:36:47,054 | DEBUG    | plugin_helper.py     (68) | loadPlugins | Total plugins loaded from basedir /home/lovesh/.sovrin are : 1
2016-11-03 21:36:47,054 | DEBUG    | client.py            (148) | __init__ | total plugins loaded in client: 1
2016-11-03 21:36:47,054 | DEBUG    | selector_events.py   (53) | __init__ | Using selector: EpollSelector
2016-11-03 21:36:47,055 | DEBUG    | looper.py            (121) | __init__ | Setting handler for SIGINT
2016-11-03 21:36:47,055 | DEBUG    | motor.py             (34) | set_status | Claaaa7 changing status from stopped to starting
2016-11-03 21:36:47,055 | INFO     | stacked.py           (75) | start | stack Claaaa7 starting at ('0.0.0.0', 5816) in always mode
2016-11-03 21:36:47,055 | INFO     | stacked.py           (851) | start | Claaaa7 listening for other nodes at 0.0.0.0:5816
2016-11-03 21:36:47,056 | DEBUG    | stacked.py           (625) | reconcileNodeReg | Claaaa7 nodereg is odict_items([('EvernymV1C', HA(host='52.33.22.91', port=9722)), ('EvernymV2C', HA(host='52.38.24.189', port=9724)), ('WSECUC', HA(host='13.93.218.255', port=9726)), ('BIGC', HA(host='52.160.103.164', port=9728)), ('RespectNetworkC', HA(host='54.173.9.185', port=9730)), ('FCUC', HA(host='69.144.146.171', port=9734)), ('BYUC', HA(host='54.193.6.155', port=9745))])
2016-11-03 21:36:47,056 | DEBUG    | stacked.py           (627) | reconcileNodeReg | Claaaa7 nodestack is []
2016-11-03 21:36:47,056 | DEBUG    | stacked.py           (504) | connectToMissing | Claaaa7 found the following missing connections: BIGC, FCUC, EvernymV2C, RespectNetworkC, BYUC, EvernymV1C, WSECUC
2016-11-03 21:36:47,186 | INFO     | stacked.py           (456) | connect | Claaaa7 looking for BIGC at 52.160.103.164:9728
2016-11-03 21:36:47,628 | INFO     | stacked.py           (456) | connect | Claaaa7 looking for FCUC at 69.144.146.171:9734
2016-11-03 21:36:47,683 | INFO     | stacked.py           (456) | connect | Claaaa7 looking for EvernymV2C at 52.38.24.189:9724
2016-11-03 21:36:47,739 | INFO     | stacked.py           (456) | connect | Claaaa7 looking for RespectNetworkC at 54.173.9.185:9730
2016-11-03 21:36:47,758 | INFO     | stacked.py           (456) | connect | Claaaa7 looking for BYUC at 54.193.6.155:9745
2016-11-03 21:36:47,812 | INFO     | stacked.py           (456) | connect | Claaaa7 looking for EvernymV1C at 52.33.22.91:9722
2016-11-03 21:36:47,832 | INFO     | stacked.py           (456) | connect | Claaaa7 looking for WSECUC at 13.93.218.255:9726
2016-11-03 21:36:47,833 | DEBUG    | stacked.py           (490) | maintainConnections | Claaaa7 next check for retries in 6.00 seconds
2016-11-03 21:36:47,838 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 0 / 7 nodes
Staler 'Claaaa7'. Stale transaction '1' packet 'None' from '('54.193.6.155', 9745)' in 1 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '0'
Staler 'Claaaa7'. Do Nack of stale correspondent ('54.193.6.155', 9745) in 1 at 1.8910829960004776
2016-11-03 21:36:48,248 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 0 / 7 nodes
Staler 'Claaaa7'. Stale transaction '1' packet 'None' from '('54.173.9.185', 9730)' in 1 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '0'
Staler 'Claaaa7'. Do Nack of stale correspondent ('54.173.9.185', 9730) in 1 at 1.9522527310036821
Staler 'Claaaa7'. Stale transaction '1' packet 'None' from '('52.33.22.91', 9722)' in 1 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '0'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.33.22.91', 9722) in 1 at 1.9522527310036821
Staler 'Claaaa7'. Stale transaction '1' packet 'None' from '('52.38.24.189', 9724)' in 1 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '0'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.38.24.189', 9724) in 1 at 1.9522527310036821
Staler 'Claaaa7'. Stale transaction '1' packet 'None' from '('52.33.22.91', 9722)' in 1 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '0'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.33.22.91', 9722) in 1 at 2.1620325750045595
Staler 'Claaaa7'. Stale transaction '1' packet 'None' from '('52.38.24.189', 9724)' in 1 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '0'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.38.24.189', 9724) in 1 at 2.180290342999797
2016-11-03 21:36:48,624 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 0 / 7 nodes
2016-11-03 21:36:48,770 | DEBUG    | stacked.py           (294) | conns | Claaaa7's connection changed from set() to {'BYUC'}
2016-11-03 21:36:48,770 | INFO     | stacked.py           (320) | _connsChanged | Claaaa7 now connected to BYUC
2016-11-03 21:36:48,771 | TRACE    | stacked.py           (775) | flushOutBoxes | Claaaa7 sending msg OrderedDict([('op', 'LEDGER_STATUS'), ('ledgerType', 0), ('txnSeqNo', 7), ('merkleRoot', 'O+RcXjejV+qOWuxtS2jn+6QGSrZm7B52Cy87P2UBOik=')]) to BYUC
2016-11-03 21:36:48,833 | DEBUG    | stacked.py           (294) | conns | Claaaa7's connection changed from {'BYUC'} to {'RespectNetworkC', 'BYUC'}
2016-11-03 21:36:48,834 | INFO     | stacked.py           (320) | _connsChanged | Claaaa7 now connected to RespectNetworkC
2016-11-03 21:36:48,834 | TRACE    | stacked.py           (775) | flushOutBoxes | Claaaa7 sending msg OrderedDict([('op', 'LEDGER_STATUS'), ('ledgerType', 0), ('txnSeqNo', 7), ('merkleRoot', 'O+RcXjejV+qOWuxtS2jn+6QGSrZm7B52Cy87P2UBOik=')]) to RespectNetworkC
2016-11-03 21:36:48,899 | DEBUG    | stacked.py           (294) | conns | Claaaa7's connection changed from {'RespectNetworkC', 'BYUC'} to {'BYUC', 'RespectNetworkC', 'EvernymV2C'}
2016-11-03 21:36:48,899 | INFO     | stacked.py           (320) | _connsChanged | Claaaa7 now connected to EvernymV2C
2016-11-03 21:36:48,899 | DEBUG    | motor.py             (34) | set_status | Claaaa7 changing status from starting to started_hungry
2016-11-03 21:36:48,899 | TRACE    | stacked.py           (775) | flushOutBoxes | Claaaa7 sending msg OrderedDict([('op', 'LEDGER_STATUS'), ('ledgerType', 0), ('txnSeqNo', 7), ('merkleRoot', 'O+RcXjejV+qOWuxtS2jn+6QGSrZm7B52Cy87P2UBOik=')]) to EvernymV2C
2016-11-03 21:36:48,960 | DEBUG    | stacked.py           (294) | conns | Claaaa7's connection changed from {'BYUC', 'RespectNetworkC', 'EvernymV2C'} to {'BYUC', 'RespectNetworkC', 'EvernymV1C', 'EvernymV2C'}
2016-11-03 21:36:48,960 | INFO     | stacked.py           (320) | _connsChanged | Claaaa7 now connected to EvernymV1C
2016-11-03 21:36:48,961 | TRACE    | stacked.py           (775) | flushOutBoxes | Claaaa7 sending msg OrderedDict([('op', 'LEDGER_STATUS'), ('ledgerType', 0), ('txnSeqNo', 7), ('merkleRoot', 'O+RcXjejV+qOWuxtS2jn+6QGSrZm7B52Cy87P2UBOik=')]) to EvernymV1C
Staler 'Claaaa7'. Stale transaction '3' packet '1' from '('54.193.6.155', 9745)' in 2 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '436'
Staler 'Claaaa7'. Do Nack of stale correspondent ('54.193.6.155', 9745) in 2 at 2.6927142440035823
Staler 'Claaaa7'. Stale transaction '1' packet 'None' from '('69.144.146.171', 9734)' in 1 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '0'
Staler 'Claaaa7'. Do Nack of stale correspondent ('69.144.146.171', 9734) in 1 at 2.6927142440035823
2016-11-03 21:36:49,022 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 4 / 7 nodes
Staler 'Claaaa7'. Stale transaction '3' packet '1' from '('54.173.9.185', 9730)' in 2 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '475'
Staler 'Claaaa7'. Do Nack of stale correspondent ('54.173.9.185', 9730) in 2 at 2.7278021890015225
Staler 'Claaaa7'. Stale transaction '1' packet 'None' from '('69.144.146.171', 9734)' in 1 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '0'
Staler 'Claaaa7'. Do Nack of stale correspondent ('69.144.146.171', 9734) in 1 at 2.795878543998697
Staler 'Claaaa7'. Stale transaction '3' packet '1' from '('52.38.24.189', 9724)' in 2 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '484'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.38.24.189', 9724) in 2 at 2.8384131400016486
Staler 'Claaaa7'. Stale transaction '3' packet '1' from '('52.33.22.91', 9722)' in 2 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '468'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.33.22.91', 9722) in 2 at 2.858899917002418
2016-11-03 21:36:49,176 | DEBUG    | client.py            (249) | handleOneNodeMsg | Client Claaaa7 got msg from node BYUC: {'op': 'CONSISTENCY_PROOF', 'ledgerType': 0, 'seqNoStart': 7, 'seqNoEnd': 8, 'oldMerkleRoot': 'Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', 'newMerkleRoot': '15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', 'hashes': ['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']}
2016-11-03 21:36:49,176 | DEBUG    | ledger_manager.py    (310) | processConsistencyProof | Claaaa7 received consistency proof: CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']) from BYUC
2016-11-03 21:36:49,176 | DEBUG    | ledger_manager.py    (591) | canStartCatchUpProcess | Claaaa7 cannot start catchup since received only 1 consistency proofs but need at least 3
2016-11-03 21:36:49,256 | DEBUG    | client.py            (249) | handleOneNodeMsg | Client Claaaa7 got msg from node RespectNetworkC: {'op': 'CONSISTENCY_PROOF', 'ledgerType': 0, 'seqNoStart': 7, 'seqNoEnd': 8, 'oldMerkleRoot': 'Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', 'newMerkleRoot': '15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', 'hashes': ['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']}
2016-11-03 21:36:49,256 | DEBUG    | ledger_manager.py    (310) | processConsistencyProof | Claaaa7 received consistency proof: CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']) from RespectNetworkC
2016-11-03 21:36:49,257 | DEBUG    | has_action_queue.py  (30) | _schedule | Claaaa7 scheduling action functools.partial(<bound method LedgerManager.checkIfCPsNeeded of Claaaa7>, 0) with id 1 to run in 30 seconds
2016-11-03 21:36:49,257 | DEBUG    | ledger_manager.py    (591) | canStartCatchUpProcess | Claaaa7 cannot start catchup since received only 2 consistency proofs but need at least 3
Staler 'Claaaa7'. Stale transaction '4' packet '1' from '('54.193.6.155', 9745)' in 3 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '436'
Staler 'Claaaa7'. Do Nack of stale correspondent ('54.193.6.155', 9745) in 3 at 3.0237555040002917
Staler 'Claaaa7'. Stale transaction '4' packet '1' from '('54.173.9.185', 9730)' in 3 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '475'
Staler 'Claaaa7'. Do Nack of stale correspondent ('54.173.9.185', 9730) in 3 at 3.0406466060012463
2016-11-03 21:36:49,398 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 4 / 7 nodes
2016-11-03 21:36:49,418 | DEBUG    | client.py            (249) | handleOneNodeMsg | Client Claaaa7 got msg from node EvernymV2C: {'op': 'CONSISTENCY_PROOF', 'ledgerType': 0, 'seqNoStart': 7, 'seqNoEnd': 8, 'oldMerkleRoot': 'Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', 'newMerkleRoot': '15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', 'hashes': ['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']}
2016-11-03 21:36:49,418 | DEBUG    | ledger_manager.py    (310) | processConsistencyProof | Claaaa7 received consistency proof: CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']) from EvernymV2C
2016-11-03 21:36:49,418 | DEBUG    | ledger_manager.py    (576) | canStartCatchUpProcess | Claaaa7 deciding on the basis of CPs {'EvernymV2C': CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']), 'RespectNetworkC': CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']), 'BYUC': CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY='])} and f 1
2016-11-03 21:36:49,418 | DEBUG    | ledger_manager.py    (656) | startCatchUpProcess | Claaaa7 started catching up with consistency proof CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=('Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY='))
2016-11-03 21:36:49,419 | DEBUG    | has_action_queue.py  (30) | _schedule | Claaaa7 scheduling action functools.partial(<bound method LedgerManager.checkIfTxnsNeeded of Claaaa7>, 0) with id 2 to run in 5.1 seconds
2016-11-03 21:36:49,419 | TRACE    | stacked.py           (775) | flushOutBoxes | Claaaa7 sending msg OrderedDict([('op', 'CATCHUP_REQ'), ('ledgerType', 0), ('seqNoStart', 8), ('seqNoEnd', 8)]) to BYUC
2016-11-03 21:36:49,472 | DEBUG    | client.py            (249) | handleOneNodeMsg | Client Claaaa7 got msg from node EvernymV1C: {'op': 'CONSISTENCY_PROOF', 'ledgerType': 0, 'seqNoStart': 7, 'seqNoEnd': 8, 'oldMerkleRoot': 'Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', 'newMerkleRoot': '15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', 'hashes': ['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']}
2016-11-03 21:36:49,472 | DEBUG    | ledger_manager.py    (310) | processConsistencyProof | Claaaa7 received consistency proof: CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']) from EvernymV1C
2016-11-03 21:36:49,472 | DEBUG    | ledger_manager.py    (344) | canProcessConsistencyProof | Claaaa7 cannot process consistency proof since in state 2 and canSync is True
2016-11-03 21:36:49,539 | DEBUG    | stacked.py           (294) | conns | Claaaa7's connection changed from {'BYUC', 'RespectNetworkC', 'EvernymV1C', 'EvernymV2C'} to {'BYUC', 'RespectNetworkC', 'FCUC', 'EvernymV1C', 'EvernymV2C'}
2016-11-03 21:36:49,539 | INFO     | stacked.py           (320) | _connsChanged | Claaaa7 now connected to FCUC
2016-11-03 21:36:49,539 | TRACE    | stacked.py           (775) | flushOutBoxes | Claaaa7 sending msg OrderedDict([('op', 'LEDGER_STATUS'), ('ledgerType', 0), ('txnSeqNo', 7), ('merkleRoot', 'O+RcXjejV+qOWuxtS2jn+6QGSrZm7B52Cy87P2UBOik=')]) to FCUC
Staler 'Claaaa7'. Stale transaction '4' packet '1' from '('52.33.22.91', 9722)' in 3 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '468'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.33.22.91', 9722) in 3 at 3.301269688003231
Staler 'Claaaa7'. Stale transaction '4' packet '1' from '('52.38.24.189', 9724)' in 3 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '484'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.38.24.189', 9724) in 3 at 3.3225067719977233
2016-11-03 21:36:49,679 | DEBUG    | client.py            (249) | handleOneNodeMsg | Client Claaaa7 got msg from node BYUC: {'op': 'CATCHUP_REP', 'ledgerType': 0, 'txns': {'8': {'reqId': 1478156921805694, 'identifier': 'CECeGXDi6EHuhpwz19uyjjEnsRGNXodFYqCRgdLmLRkt', 'type': 'CHANGE_HA', 'txnTime': 1478157003369.697, 'data': {'node_ip': '40.121.149.41', 'alias': 'WSECU', 'client_port': 9726, 'client_ip': '40.121.149.41', 'node_port': 9725}, 'txnId': 'bfa9e020f5963c12132024d5a2067c2bd6f53b5244cf247e9e50872c9249704b', 'dest': 'AZsp46qSPGjHuNYajiMUb5XcMWCRDuk2xdEv7P5MepSN'}}, 'consProof': []}
2016-11-03 21:36:49,680 | DEBUG    | ledger_manager.py    (384) | processCatchupRep | Claaaa7 received catchup reply from BYUC: CATCHUP_REP(ledgerType=0, txns={'8': {'reqId': 1478156921805694, 'identifier': 'CECeGXDi6EHuhpwz19uyjjEnsRGNXodFYqCRgdLmLRkt', 'type': 'CHANGE_HA', 'txnTime': 1478157003369.697, 'data': {'node_ip': '40.121.149.41', 'alias': 'WSECU', 'client_port': 9726, 'client_ip': '40.121.149.41', 'node_port': 9725}, 'txnId': 'bfa9e020f5963c12132024d5a2067c2bd6f53b5244cf247e9e50872c9249704b', 'dest': 'AZsp46qSPGjHuNYajiMUb5XcMWCRDuk2xdEv7P5MepSN'}}, consProof=[])
2016-11-03 21:36:49,680 | DEBUG    | ledger_manager.py    (390) | processCatchupRep | Claaaa7 found 1 transactions in the catchup from BYUC
2016-11-03 21:36:49,680 | DEBUG    | ledger_manager.py    (399) | processCatchupRep | Claaaa7 merging all received catchups
2016-11-03 21:36:49,680 | DEBUG    | ledger_manager.py    (405) | processCatchupRep | Claaaa7 merged catchups, there are 1 of them now, from 8 to 8
2016-11-03 21:36:49,680 | DEBUG    | ledger_manager.py    (496) | hasValidCatchupReplies | Claaaa7 verifying proof for 8, 8, b'\xc8v\xe37\xc5\xb5\xf0S\x827m\x9d\xe9\xbf\x8b\xc9\xc4\xd7\xaf)\x0b\xb5\xfa\x06\x90\xb1Ed\xb8\xb9ya', b'\xd7\x99p\xfar\xf5\xd9\x9e\xb8\xd2\x05\xe3\x81\x9f\x8c\xdd2;\xfd\xab(\xf6\xbcU\tNW\xc0jDMV', []
2016-11-03 21:36:49,681 | INFO     | ledger_manager.py    (505) | hasValidCatchupReplies | Claaaa7 could not verify catchup reply CATCHUP_REP(ledgerType=0, txns={'8': {'reqId': 1478156921805694, 'identifier': 'CECeGXDi6EHuhpwz19uyjjEnsRGNXodFYqCRgdLmLRkt', 'type': 'CHANGE_HA', 'txnTime': 1478157003369.697, 'data': {'node_ip': '40.121.149.41', 'alias': 'WSECU', 'client_port': 9726, 'client_ip': '40.121.149.41', 'node_port': 9725}, 'txnId': 'bfa9e020f5963c12132024d5a2067c2bd6f53b5244cf247e9e50872c9249704b', 'dest': 'AZsp46qSPGjHuNYajiMUb5XcMWCRDuk2xdEv7P5MepSN'}}, consProof=[]) since Inconsistency: different root hashes for the same tree size
2016-11-03 21:36:49,681 | DEBUG    | ledger_manager.py    (413) | processCatchupRep | Claaaa7 processed 0 catchup replies with sequence numbers []
2016-11-03 21:36:49,775 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
Staler 'Claaaa7'. Stale transaction '3' packet '1' from '('69.144.146.171', 9734)' in 2 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '337'
Staler 'Claaaa7'. Do Nack of stale correspondent ('69.144.146.171', 9734) in 2 at 3.471773110999493
2016-11-03 21:36:49,878 | DEBUG    | client.py            (249) | handleOneNodeMsg | Client Claaaa7 got msg from node FCUC: {'op': 'CONSISTENCY_PROOF', 'ledgerType': 0, 'seqNoStart': 7, 'seqNoEnd': 8, 'oldMerkleRoot': 'Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', 'newMerkleRoot': '15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', 'hashes': ['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']}
2016-11-03 21:36:49,879 | DEBUG    | ledger_manager.py    (310) | processConsistencyProof | Claaaa7 received consistency proof: CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']) from FCUC
2016-11-03 21:36:49,879 | DEBUG    | ledger_manager.py    (344) | canProcessConsistencyProof | Claaaa7 cannot process consistency proof since in state 2 and canSync is True
Staler 'Claaaa7'. Stale transaction '4' packet '1' from '('69.144.146.171', 9734)' in 3 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '337'
Staler 'Claaaa7'. Do Nack of stale correspondent ('69.144.146.171', 9734) in 3 at 3.789192272000946
2016-11-03 21:36:50,151 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:50,527 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:50,903 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
Staler 'Claaaa7'. Stale transaction '1' packet 'None' from '('52.160.103.164', 9728)' in 1 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '0'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.160.103.164', 9728) in 1 at 4.698988483003632
2016-11-03 21:36:51,280 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:51,656 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:52,033 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:52,410 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:52,787 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:53,056 | TRACE    | stacked.py           (542) | handleDisconnectedRemote | Claaaa7 allow already in process, so waiting to check for reconnects
2016-11-03 21:36:53,057 | TRACE    | stacked.py           (534) | handleDisconnectedRemote | Claaaa7 join already in process, so waiting to check for reconnects
2016-11-03 21:36:53,057 | DEBUG    | stacked.py           (625) | reconcileNodeReg | Claaaa7 nodereg is odict_items([('EvernymV1C', HA(host='52.33.22.91', port=9722)), ('EvernymV2C', HA(host='52.38.24.189', port=9724)), ('WSECUC', HA(host='13.93.218.255', port=9726)), ('BIGC', HA(host='52.160.103.164', port=9728)), ('RespectNetworkC', HA(host='54.173.9.185', port=9730)), ('FCUC', HA(host='69.144.146.171', port=9734)), ('BYUC', HA(host='54.193.6.155', port=9745))])
2016-11-03 21:36:53,057 | DEBUG    | stacked.py           (627) | reconcileNodeReg | Claaaa7 nodestack is ['BIGC', 'FCUC', 'EvernymV2C', 'RespectNetworkC', 'BYUC', 'EvernymV1C', 'estate_857b1568a1df11e6ac2c6807152028e4']
2016-11-03 21:36:53,057 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 2 ('52.160.103.164', 9728)
2016-11-03 21:36:53,057 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 3 ('69.144.146.171', 9734)
2016-11-03 21:36:53,057 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 4 ('52.38.24.189', 9724)
2016-11-03 21:36:53,057 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 5 ('54.173.9.185', 9730)
2016-11-03 21:36:53,057 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 6 ('54.193.6.155', 9745)
2016-11-03 21:36:53,057 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 7 ('52.33.22.91', 9722)
2016-11-03 21:36:53,057 | DEBUG    | stacked.py           (651) | reconcileNodeReg | Claaaa7 unmatched remote is 8 ('13.93.218.255', 9726)
2016-11-03 21:36:53,058 | DEBUG    | stacked.py           (655) | reconcileNodeReg | Claaaa7 forgiving name mismatch for WSECUC with same ha ('13.93.218.255', 9726) using another name estate_857b1568a1df11e6ac2c6807152028e4
2016-11-03 21:36:53,058 | DEBUG    | stacked.py           (490) | maintainConnections | Claaaa7 next check for retries in 6.00 seconds
2016-11-03 21:36:53,164 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:53,541 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:53,916 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 5 / 7 nodes
2016-11-03 21:36:54,016 | DEBUG    | stacked.py           (294) | conns | Claaaa7's connection changed from {'BYUC', 'RespectNetworkC', 'FCUC', 'EvernymV1C', 'EvernymV2C'} to {'FCUC', 'BYUC', 'EvernymV2C', 'EvernymV1C', 'BIGC', 'RespectNetworkC'}
2016-11-03 21:36:54,016 | INFO     | stacked.py           (320) | _connsChanged | Claaaa7 now connected to BIGC
2016-11-03 21:36:54,016 | TRACE    | stacked.py           (775) | flushOutBoxes | Claaaa7 sending msg OrderedDict([('op', 'LEDGER_STATUS'), ('ledgerType', 0), ('txnSeqNo', 7), ('merkleRoot', 'O+RcXjejV+qOWuxtS2jn+6QGSrZm7B52Cy87P2UBOik=')]) to BIGC
2016-11-03 21:36:54,293 | DEBUG    | gen_load_ratcheting  (216) | checkIfConnectedToAll | Connected 6 / 7 nodes
2016-11-03 21:36:54,293 | DEBUG    | eventually.py        (86) | eventuallyAll | partial(checkIfConnectedToAll) succeeded with 1.05 seconds to spare
2016-11-03 21:36:54,294 | TRACE    | signing.py           (86) | serializeMsg | serialized msg {'signature': None, 'identifier': 'BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex', 'operation': {'type': 'ATTRIB', 'raw': '0c703bdd74ae5d4fbea61591b06fcc40034c1072ed190fa365c89a1eba030886'}, 'reqId': 1478189214294136} into identifier:BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex|operation:raw:0c703bdd74ae5d4fbea61591b06fcc40034c1072ed190fa365c89a1eba030886|type:ATTRIB|reqId:1478189214294136
2016-11-03 21:36:54,295 | DEBUG    | client.py            (228) | submitReqs | Claaaa7 pending request since in mode 1 and connected to {'FCUC', 'BYUC', 'EvernymV2C', 'EvernymV1C', 'BIGC', 'RespectNetworkC'} nodes
2016-11-03 21:36:54,295 | DEBUG    | client.py            (416) | pendReqsTillConnection | Enqueuing request since not enough connections with nodes: Request: {'identifier': 'BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex', 'operation': {'type': 'ATTRIB', 'raw': '{"mD5PfgQvpSXtKwdB3h0l_462804": "919812841765_8QJT9unBPwV73NI1DStHedG2LUmxKgqAca0oOlC4jX5ZfR6Fsz"}'}, 'reqId': 1478189214294136, 'signature': '3FGAuZ5aRRErTgKpoZdXWqiqmDKsuwQjPK2Kjo588cKLMTkZ9cZyZ7e3ghc9b4ZTYJzsr2WzPTnMPGPun4GdRG2R'}
2016-11-03 21:36:54,295 | DEBUG    | gen_load_ratcheting  (123) | _submit | Client Claaaa7 sending request Request: {'identifier': 'BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex', 'operation': {'type': 'ATTRIB', 'raw': '{"mD5PfgQvpSXtKwdB3h0l_462804": "919812841765_8QJT9unBPwV73NI1DStHedG2LUmxKgqAca0oOlC4jX5ZfR6Fsz"}'}, 'reqId': 1478189214294136, 'signature': '3FGAuZ5aRRErTgKpoZdXWqiqmDKsuwQjPK2Kjo588cKLMTkZ9cZyZ7e3ghc9b4ZTYJzsr2WzPTnMPGPun4GdRG2R'}
2016-11-03 21:36:54,529 | DEBUG    | has_action_queue.py  (60) | _serviceActions | Claaaa7 running action functools.partial(<bound method LedgerManager.checkIfTxnsNeeded of Claaaa7>, 0) with id 2
Staler 'Claaaa7'. Stale transaction '3' packet '1' from '('52.160.103.164', 9728)' in 2 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '477'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.160.103.164', 9728) in 2 at 9.183705485003884
Staler 'Claaaa7'. Stale transaction '4' packet '1' from '('52.160.103.164', 9728)' in 3 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '477'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.160.103.164', 9728) in 3 at 11.235203044998343
Staler 'Claaaa7'. Stale transaction '4' packet '1' from '('52.160.103.164', 9728)' in 3 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '477'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.160.103.164', 9728) in 3 at 11.235203044998343
2016-11-03 21:36:59,067 | TRACE    | stacked.py           (534) | handleDisconnectedRemote | Claaaa7 join already in process, so waiting to check for reconnects
2016-11-03 21:36:59,067 | DEBUG    | stacked.py           (625) | reconcileNodeReg | Claaaa7 nodereg is odict_items([('EvernymV1C', HA(host='52.33.22.91', port=9722)), ('EvernymV2C', HA(host='52.38.24.189', port=9724)), ('WSECUC', HA(host='13.93.218.255', port=9726)), ('BIGC', HA(host='52.160.103.164', port=9728)), ('RespectNetworkC', HA(host='54.173.9.185', port=9730)), ('FCUC', HA(host='69.144.146.171', port=9734)), ('BYUC', HA(host='54.193.6.155', port=9745))])
2016-11-03 21:36:59,067 | DEBUG    | stacked.py           (627) | reconcileNodeReg | Claaaa7 nodestack is ['BIGC', 'FCUC', 'EvernymV2C', 'RespectNetworkC', 'BYUC', 'EvernymV1C', 'estate_857b1568a1df11e6ac2c6807152028e4']
2016-11-03 21:36:59,067 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 2 ('52.160.103.164', 9728)
2016-11-03 21:36:59,068 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 3 ('69.144.146.171', 9734)
2016-11-03 21:36:59,068 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 4 ('52.38.24.189', 9724)
2016-11-03 21:36:59,068 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 5 ('54.173.9.185', 9730)
2016-11-03 21:36:59,068 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 6 ('54.193.6.155', 9745)
2016-11-03 21:36:59,068 | DEBUG    | stacked.py           (633) | reconcileNodeReg | Claaaa7 matched remote is 7 ('52.33.22.91', 9722)
2016-11-03 21:36:59,068 | DEBUG    | stacked.py           (651) | reconcileNodeReg | Claaaa7 unmatched remote is 8 ('13.93.218.255', 9726)
2016-11-03 21:36:59,068 | DEBUG    | stacked.py           (655) | reconcileNodeReg | Claaaa7 forgiving name mismatch for WSECUC with same ha ('13.93.218.255', 9726) using another name estate_857b1568a1df11e6ac2c6807152028e4
2016-11-03 21:36:59,068 | DEBUG    | stacked.py           (490) | maintainConnections | Claaaa7 next check for retries in 6.00 seconds
2016-11-03 21:36:59,330 | DEBUG    | gen_load_ratcheting  (206) | checkReplyAndLogStat | COUNTER ResultRow(signerName='Claaaa7', signerId='BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex', reqId=1478189214294136, transactionType='ATTRIB', sentAt=1478189214.2949615, quorumAt='', latency=5.035929203033447, ackNodes='', nackNodes='', replyNodes='')
2016-11-03 21:36:59,331 | DEBUG    | eventually.py        (86) | eventuallyAll | partial(checkReplyAndLogStat) succeeded with 2.46 seconds to spare
2016-11-03 21:36:59,331 | DEBUG    | gen_load_ratcheting  (269) | printCurrentTestResults | 
        ================================
        Test time: 5.038129568099976
        Average latency: 5.035929203033447
        Throughput: 0.0
        Error rate: 0.19848636016265236
        Succeeded: 0
        Failed: 1
        Query Time: 0
        ================================
        
2016-11-03 21:36:59,332 | TRACE    | signing.py           (86) | serializeMsg | serialized msg {'signature': None, 'identifier': 'BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex', 'operation': {'type': 'ATTRIB', 'raw': 'f952cc27d5f1a7e8802e7bbe955b8c8023b2aeb4bd281d2ce9b0f3eecca16d14'}, 'reqId': 1478189219332340} into identifier:BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex|operation:raw:f952cc27d5f1a7e8802e7bbe955b8c8023b2aeb4bd281d2ce9b0f3eecca16d14|type:ATTRIB|reqId:1478189219332340
2016-11-03 21:36:59,332 | DEBUG    | client.py            (228) | submitReqs | Claaaa7 pending request since in mode 1 and connected to {'FCUC', 'BYUC', 'EvernymV2C', 'EvernymV1C', 'BIGC', 'RespectNetworkC'} nodes
2016-11-03 21:36:59,333 | DEBUG    | client.py            (416) | pendReqsTillConnection | Enqueuing request since not enough connections with nodes: Request: {'identifier': 'BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex', 'operation': {'type': 'ATTRIB', 'raw': '{"ivqcokKFs0DY3SUR6luG_751141": "617249405812_7i9N3oGTOncl4QuWyvZDtBLU5mMAkVqIaYEb0dH8rghXSp2CeP"}'}, 'reqId': 1478189219332340, 'signature': '5wN7D67TciW5sUQHDjdLHfqKayPfSfpjYDBTo8bpPenGmWs5SUooQ6ErGATwqpcNKXM2U7zeJYpUJRTCD76gwF3T'}
2016-11-03 21:36:59,333 | DEBUG    | gen_load_ratcheting  (123) | _submit | Client Claaaa7 sending request Request: {'identifier': 'BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex', 'operation': {'type': 'ATTRIB', 'raw': '{"ivqcokKFs0DY3SUR6luG_751141": "617249405812_7i9N3oGTOncl4QuWyvZDtBLU5mMAkVqIaYEb0dH8rghXSp2CeP"}'}, 'reqId': 1478189219332340, 'signature': '5wN7D67TciW5sUQHDjdLHfqKayPfSfpjYDBTo8bpPenGmWs5SUooQ6ErGATwqpcNKXM2U7zeJYpUJRTCD76gwF3T'}
Staler 'Claaaa7'. Stale transaction '4' packet '1' from '('52.160.103.164', 9728)' in 3 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '477'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.160.103.164', 9728) in 3 at 14.012059307999152
Staler 'Claaaa7'. Stale transaction '4' packet '1' from '('52.160.103.164', 9728)' in 3 nacking...
Staler 'Claaaa7'. Unknown correspondent estate id '477'
Staler 'Claaaa7'. Do Nack of stale correspondent ('52.160.103.164', 9728) in 3 at 14.012059307999152
2016-11-03 21:37:00,375 | DEBUG    | client.py            (249) | handleOneNodeMsg | Client Claaaa7 got msg from node BIGC: {'op': 'CONSISTENCY_PROOF', 'ledgerType': 0, 'seqNoStart': 7, 'seqNoEnd': 8, 'oldMerkleRoot': 'Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', 'newMerkleRoot': '15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', 'hashes': ['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']}
2016-11-03 21:37:00,376 | DEBUG    | ledger_manager.py    (310) | processConsistencyProof | Claaaa7 received consistency proof: CONSISTENCY_PROOF(ledgerType=0, seqNoStart=7, seqNoEnd=8, oldMerkleRoot='Ym2M5MCcYtL9Nhl+JinisTVwK4iCDbU+mcKedj+pYLY=', newMerkleRoot='15lw+nL12Z640gXjgZ+M3TI7/aso9rxVCU5XwGpETVY=', hashes=['Kk5CspFcrPDCFrch/iflg1tXS3Htv3H7o6vUYeY9m0U=', 'gpkiCCgaoZdRd6QBalKTGMK4WsVMHGDirI8tH3DCGG8=', 'qNa3BH7CvBHjSAJJ+BXqzy1Bd3aDInGyB9IeS5fpCdI=', 'tXipkaSw3qcSHcCk8LEqSxd0UWgELv40mIs8OZgCImY=']) from BIGC
2016-11-03 21:37:00,376 | DEBUG    | ledger_manager.py    (344) | canProcessConsistencyProof | Claaaa7 cannot process consistency proof since in state 2 and canSync is True
Stalent 'Claaaa7'. Stale transaction '0' packet '0' from '('52.160.103.164', 9728)' in 1 nacking ...
Stalent 'Claaaa7'. Unknown initiator estate id '477'
Stalent 'Claaaa7'. Do Nack of ('52.160.103.164', 9728) in 1 at 15.210290501003328
2016-11-03 21:37:04,404 | DEBUG    | gen_load_ratcheting  (206) | checkReplyAndLogStat | COUNTER ResultRow(signerName='Claaaa7', signerId='BXXbxdL3V4FzYVVLWNYjrGtyFquMyVda7zrQEPLZ5eex', reqId=1478189219332340, transactionType='ATTRIB', sentAt=1478189219.3328621, quorumAt='', latency=5.071848154067993, ackNodes='', nackNodes='', replyNodes='')
2016-11-03 21:37:04,405 | DEBUG    | eventually.py        (86) | eventuallyAll | partial(checkReplyAndLogStat) succeeded with 2.43 seconds to spare
2016-11-03 21:37:04,405 | DEBUG    | gen_load_ratcheting  (269) | printCurrentTestResults | 
        ================================
        Test time: 10.111500263214111
        Average latency: 5.05388867855072
        Throughput: 0.0
        Error rate: 0.1977945851691316
        Succeeded: 0
        Failed: 2
        Query Time: 0
        ================================
        
2016-11-03 21:37:04,405 | DEBUG    | gen_load_ratcheting  (326) | main | Sent 1 ATTRIB requests
2016-11-03 21:37:04,405 | INFO     | looper.py            (253) | shutdown | Looper shutting down now...
2016-11-03 21:37:04,417 | DEBUG    | motor.py             (34) | set_status | Claaaa7 changing status from started_hungry to stopping
2016-11-03 21:37:04,417 | INFO     | stacked.py           (82) | stop | stack Claaaa7 stopped
2016-11-03 21:37:04,417 | DEBUG    | motor.py             (34) | set_status | Claaaa7 changing status from stopping to stopped
2016-11-03 21:37:04,418 | INFO     | looper.py            (260) | shutdown | Looper shut down in 0.012 seconds.
2016-11-03 21:37:04,457 | DEBUG    | base_events.py       (410) | close | Close <_UnixSelectorEventLoop running=False closed=False debug=True>
